multiaccounts/settings/database.go

```diff
@@ -744,10 +748,46 @@ func (db *Database) GetTokenGroupByCommunity() (result bool, err error) {
 	return result, err
 }
 
+func (db *Database) GetShowCommunityAssetWhenSendingTokens() (result bool, err error) {
```

Comment:
@Cuteivist at this moment no need for this and other functions which added below within this PR. Do we need/use them anyhow?